src/pages/About.jsx

```diff
@@ -48,6 +48,17 @@ const team = [
       </>
     ),
   },
+    {
+        name: 'Job Hernandez',
+        img: testPic,
+        description: (
+            <>
+            I am interested and driven to solve problems with software; software
+            has tremendous power to make an impact in the world and transform lives.
+               </>
+        ),
+    },
+        
```

Comment:
The [style check](https://github.com/beyondhb1079/s4us/pull/349/checks?check_run_id=1774597880) is failing due to this not being properly formatted. This is auto-formatted when you do `git commit` from the command line so I imagine this must've been edited directly on GitHub. You'll need to make the indentation match up with the other entries. So,

```suggestion
  {
    name: 'Job Hernandez',
    img: testPic,
    description: (
      <>
        I am interested and driven to solve problems with software; software
        has tremendous power to make an impact in the world and transform lives.
      </>
    ),
  },
        
```